src/commands/data/connectors/update.ts

```diff
@@ -33,12 +33,40 @@ export default class ConnectorsUpdate extends BaseCommand {
         return [key, value]
       },
     }),
+    addTable: flags.string({
+      char: 't',
+      description: 'Tables to add',
+      multiple: true,
+      required: false,
+    }),
+    removeTable: flags.string({
+      char: 't',
```

Comment:
I don't think we can use the same `char` for more than one command. I've seen conventions that use lower/upper case for add/remove. Or we could skip the `char` shortcuts entirely.

Reply:
Good catch. I am okay with skipping them cause there are quite a few flags at this point.